portfolio/src/main/webapp/script.js

```diff
@@ -61,7 +61,7 @@ function getFeedback() {
 function createListElement(key, text) {
   const liElement = document.createElement('li');
   liElement.id = key;
-  liElement.innerText = text.comment + ',' + text.userEmail + '\t';
+  liElement.innerText = text.commentText + ',' + text.userEmail + '\t';
```

Comment:
Why is this change in this PR?

Reply:
Because when you change the comment to commentText, when accessing it in JS you need to change the attribute name as well